### DISCUSSION TITLE
The image is not loading when sharing an article on Facebook.

### DISCUSSION THREAD

**Discussion by dusanex** [templaza/astroid-framework#906] [General]
When I create an article and select the article type as video/YouTube, Facebook is unable to load the video thumbnail when posting. Can you solve this problem?

**Comment by paurray** [comment]
@dusanex 
I am sorry to say that I do not have an answer to this problem but I too would be very interested in a solution 👍 

<img width="960" alt="FB-Share-Problems" src="https://github.com/user-attachments/assets/25ee7e50-8240-4d4f-8cf5-5eddccb4cfa3">

**Reply by sonvnn** [reply]
Your article is shared successful here https://www.facebook.com/groups/astroid
You can check it.

**Comment by dusanex** [comment]
Interesting that the "Helix Ultimate Framework" has no issue with it – can't someone fix it?

**Comment by sonvnn** [comment]
I tried to publish a post on Facebook and it works
![Screen Shot 2024-12-10 at 08 38 41](https://github.com/user-attachments/assets/a2363655-048c-4fd3-b15f-aa9f574e890e)

Anyone else can help me to confirm this issue.
Thanks,
Sonny

**Comment by sonvnn** [comment]
@dusanex @paurray I tried to improve this on https://github.com/templaza/astroid-framework/commit/1ddf31174b73bf35f082791a3b2b8050a805ce01

Can you help me download v3-branch and retest it again? Please try with another article because your old article was cached by Facebook.

Best Regards,
Sonny

**Reply by dusanex** [reply]
Hello, thank you, but it doesn't work correctly. I noticed that the image loads only if related articles are displayed. And the image is taken from the first related article, which means that the wrong image is assigned to the post on Facebook. :(

**Reply by paurray** [reply]
@sonvnn Very busy here at the moment not sure if I can test this year. I believe that this has come up on multiple occasions in the discussions section but cant find anything right now.
If you have a chance please look at this:
https://github.com/templaza/astroid-framework/discussions/910

**Comment by dusanex** [comment]
Hi, I need this to work, could you please take a look at it and fix it? Why doesn't such a simple thing like sharing on Facebook work? It's just a YouTube video with a clear link to the video thumbnail image.

**Reply by sonvnn** [reply]
Can you send me your website link to check? I still can not reproduce your issue on my site.

**Reply by dusanex** [reply]
https://transportal.sk    
Thank you in advance for your help...

**Reply by dusanex** [reply]
![2024-12-18 22 05 31](https://github.com/user-attachments/assets/c5410fca-132a-4a8b-9650-8328566580f3)

**Comment by tmumm** [comment]
Did you try adding Open Graph info and image in the menu under Open Graph tab?

**Reply by dusanex** [reply]
Thank you for the response. Yes, I tried it, but it should work even without that. Could you check how it's resolved in the Helix Ultimate Framework? It works there...
